kubernetes.yaml

```diff
@@ -106,7 +108,7 @@ data:
     limits_memory = 0.5G
     
     [joblogs]
-    logs_dir = /tmp/scrapyd_k8s_logs
+    logs_dir = /data/scrapyd_k8s_logs
```

Comment:
let's call this `/data/joblogs` - the PVC is just for scrapyd-k8s, and the feature is called "joblogs"

Reply:
Also, shall we drop the PV below? Or at least comment that this may be needed in case x, but not in case y?